content/finding-command-help.md

```diff
@@ -0,0 +1,131 @@
+
+### Why Find Help Using the Terminal?
+
+Issues may disable user access to the desktop environment, including the web browser. In these scenarios, users can still find help that displays as output in a Terminal session. Users who still have access to a browser or second computer can find help using the [Ubuntu](https://manpages.ubuntu.com/) or [Debian](https://manpages.debian.org/) manpage websites. The commands below are entered in a Terminal session. In Pop!\_OS, Terminal sessions can be started by pressing <kbd>Super</kbd> + <kbd>t</kbd>.
+
+### --help
+
+`--help` prints out a shortened version of the man page, usually including a brief example and all options for a command. `--help` prints output to the terminal session without navigating away from the shell prompt.
+
+**Use case**: A user wants to quickly reference all options for a known command in the current Terminal session.
+
+**Usage**: Type a command + `--help`, then hit <kbd>Enter</kbd>. This example uses the `cd` command. Commands must be an exact match.
+
+```bash
+cd --help
+```
+![--help output](/images/finding-command-help/--help-output.png)
+
+### whatis
+
+`whatis` provides a very short description of a command's functionality.
+
+**Use case**: A user wants a very brief reminder of a known command's functionality printed in the current Terminal session.
+
+**Usage**: Type `whatis` + command, then hit <kbd>Enter</kbd>. This example uses the `mkdir` command. Commands must be an exact match.
+
+```bash
+whatis mkdir
+```
+![whatis output](/images/finding-command-help/what-output.png)
+
+### which & dpkg -S
+
+`which` provides the directory location of a command. The output of `which` can be passed into `dpkg -S` to perform a search of associated packages.
+
+**Use case**: A user is attempting to learn about commands included with a recently installed program, and would like to know which installed package provides a specific command  
+
+**Usage**: Type `which` + command, then pipe the output of `which` into `dpkg -S` using `xargs`. The `xargs` command allows users to pass the output of a command as standard input into another command. The below example searches for the path of the `libreoffice` command, and then searches for the program associated with that path.
+
+```bash
+which libreoffice | xargs dpkg -S
+```
+![which output](/images/finding-command-help/which-output.png)
+
+Users may see an error stating no matching path can be found. This occurs when `which` returns a symbolic link instead of the actual installation directory. Packages create symbolic links because the Terminal only recognizes executable commands from specific default and user defined directories. `dpkg` only knows the intended default installation path for a package, but may not know which package owns a symbolic link. Pass the path output from `which` into the `ls -alh` command to verify if an executable file is symbolically linked.
```

Comment:
> This occurs when `which` returns a symbolic link instead of the actual installation directory. Packages create symbolic links because the Terminal only recognizes executable commands from specific default and user defined directories.

- `which` returning a symbolic link is not the cause of this; `which` returning a symbolic link (or any file) that wasn't included in a package is the cause of this.

> Packages create symbolic links because the Terminal only recognizes executable commands from specific default and user defined directories.

This is true, but not entirely relevant to the error message at hand for the above reason.

> `dpkg` only knows the intended default installation path for a package, but may not know which package owns a symbolic link.

This is a misrepresentation of the issue. Again, if a package owns a symbolic link, `dpkg` will know which one it is. The issue is if a package installs into a directory (such as `/bin`) that had already been symlinked to another directory (`/usr/bin`) before installation.

This misunderstanding might be partially my fault for the examples I provided. For the shutdown example, I was actually unaware _what_ was creating the `/usr/sbin/shutdown` file-- I assumed it was being created by a post-install script or systemd itself, rather than being packaged. However, I did some digging and discovered that `/usr/sbin/shutdown` is actually installed to `/sbin/shutdown`, which dpkg recognizes as installed from the `systemd-sysv` package. So it's actually the same issue as the one I showed with `mkdir` earlier.